python/batch_qa/download_batch_qa_excel.py

```
@@ -0,0 +1,26 @@
+import os
+from urllib.parse import urljoin
```

Ruff

`urllib.parse.urljoin` imported but unused

Remove unused import: `urllib.parse.urljoin`

(F401)